tests/keras/test_callbacks.py

```diff
@@ -247,12 +247,12 @@ def test_EarlyStopping_reuse():
     stopper = callbacks.EarlyStopping(monitor='acc', patience=patience)
     weights = model.get_weights()
 
-    hist = model.fit(data, labels, callbacks=[stopper])
+    hist = model.fit(data, labels, callbacks=[stopper], epochs=20)
```

Comment:
This would slow down the tests and should be set to a lower value (e.g. 10). Is this change related to the rest of the PR? Doesn't look like it is...

Reply:
Without this change (introduced in the second commit) tests fail (see first commit). Maybe this change is just papering over a real bug?

I chose epochs=20, because earlier in the same function epochs=20 is used. In the whole file, epochs=20 is used 8 times, and epochs=10 is used 0 times.

Reply:
Ok then.